### ISSUE TITLE
Global components

### ISSUE THREAD

**Issue by edkf** [edkf/Sketch-hand-draw-components-template#5]
I found this components.
I think that some are useless, but i like this example.
![image](https://cloud.githubusercontent.com/assets/3325227/20968350/47b80aa4-bc6b-11e6-8f11-8f6bd98ceda8.png)

**Comment by JulienMalige** [comment]
Yes, I was just thinking of decomposing it (starting with geometric shapes for schema). Do you prefer to make a big one in one shot ?

**Comment by edkf** [comment]
I dont know lol.
I think that we need first to find a style and after start to create the components.
